eks/iam.tf

```diff
@@ -182,3 +182,58 @@ resource "aws_iam_role_policy_attachment" "fargate-AmazonEKSFargatePodExecutionR
   policy_arn = "arn:aws:iam::aws:policy/AmazonEKSFargatePodExecutionRolePolicy"
   role       = aws_iam_role.fargate.name
 }
+
+module "iam_assumable_role_admin" {
+  count                         = var.efs_enabled ? 1 : 0
+  source                        = "terraform-aws-modules/iam/aws//modules/iam-assumable-role-with-oidc"
```

Comment:
Where is this source? I don't think you added it.

Reply:
if you do `terraform init `from here it should work https://github.com/abhiyerra/Upwork/pull/3/files

Reply:
you mean to load the module from this repo itself?